lib/commands/cloud-codesign.ts

```diff
@@ -0,0 +1,50 @@
+import { isInteractive } from "../helpers";
+
+export class CloudCodesignCommand implements ICommand {
+	private parametersValidationText = "The command has only 3 valid parameters - Apple account id, Apple account password and optional parameter clean for force re-invokation of certificate, if any.";
+	public allowedParameters: ICommandParameter[];
+
+	constructor(private $logger: ILogger,
+		private $errors: IErrors,
+		private $projectData: IProjectData,
+		private $prompter: IPrompter,
+		private $options: IOptions,
+		private $cloudCodesignService: ICloudCodesignService) {
+		this.$projectData.initializeProjectData();
+	}
+
+	public async execute(args: string[]): Promise<void> {
+		let username = args[0];
+		let password = args[1];
+
+		if (!username) {
+			if (isInteractive) {
```

Comment:
You can check for `isInteractive` in `canExecute` method, as you are currently failing with the same error. For example:
```TypeScript
public async canExecute(args: string[]): Promise<boolean> {
	if (args.length > 3) {
		this.$errors.fail(this.parametersValidationText);
	}

	if (!isInteractive() && args.length < 2) {
		this.$errors.fail(this.parametersValidationText);
	}
	
	return true;
}
```